components/match2/wikis/arenaofvalor/match_group_input_custom.lua

```diff
@@ -466,7 +466,7 @@ function mapFunctions.getParticipants(map, opponents)
 		local banIndex = 1
 		local currentBan = map['t' .. opponentIndex .. 'b' .. banIndex]
 		while currentBan do
-			championData['team' .. opponentIndex .. 'ban' .. banIndex] = ChampionNames[currentBan] or currentBan
+			championData['team' .. opponentIndex .. 'ban' .. banIndex] = ChampionNames[currentBan and currentBan:lower()]
```

Comment:
removing the `or` at the end of the line intended?

Reply:
yes, all hero inputs should go though alias mapping

Reply:
maybe do an assert then to let it show an error for invalid input
this change will lead to invalid inputs just not being stored/shown

Reply:
they are just empty now

Reply:
yap :D